src/features/user/TreemapperMigration/TreemapperMigration.module.scss

```
@@ -0,0 +1,57 @@
+  background-color: $dsWhite;
+  border-radius: 24px;
+  padding: 24px;
+  height: 640px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 16px;
+  text-align: center;
+
+  background-image: url('/assets/images/treemapper/treemapper-migration-background.png');
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)